### ISSUE TITLE
Code is broken

### ISSUE THREAD

**Issue by gabrielmdc** [nearlg/gcr-server#33]
For some reason, there are some variables that their names was changed to _print_.
In _main.py_ seems like the references shuldn't have a dot in the beginning:

```python
from .lib.connection import Connection
from .lib.supervisor import SupervisorThread
from .lib.repository.repositories import Repositories
```
TO
```python
from lib.connection import Connection
from lib.supervisor import SupervisorThread
from lib.repository.repositories import Repositories
```

The file _daemon.py_ must change the line 7 to _DB_FILE = "resources/database.db"_
The file _repository/gpio.py_ must change the line 11 to _GPIO_DIRECTORY_NAME = "/sys/class/gpio"_
The function _prepare_gpios_ must change the values of _high_ and _low_ to _'1'_ and _'0'_

**Comment by gabrielmdc** [comment]
The code is not broken as is mentioned in the issue description.

**Comment by gabrielmdc** [comment]
I think is good idea to change this:

> The function prepare_gpios must change the values of high and low to '1' and '0'

In the [version of python 2](https://github.com/nearlg/gcr-server-p2)

**Comment by gabrielmdc** [comment]
Fixed #38
